internal/version.go

```diff
@@ -0,0 +1,8 @@
+package internal
+
+import "github.com/icinga/icingadb/pkg/version"
+
+// Version contains version and Git commit information.
+//
+// The placeholders are replaced on `git archive` using the `export-subst` attribute.
+var Version = version.Version("0.0.0", "$Format:%(describe)$", "$Format:%H$")
```

Comment:
Just for the record: `$Format:%(describe)$` does not work properly when there are no tags in the repository. This means that at the moment, if one actually uses `git archive` (including for example downloading the source from GitHub as a tarball), it would report an empty string as version. But that will fix itself once there's a tag in this repo.